app/views/Export.js

```diff
@@ -129,27 +130,39 @@ export const ExportScreen = ({ navigation }) => {
 
           <ScrollView contentContainerStyle={styles.contentContainer}>
             <View style={styles.main}>
-              <Typography style={styles.exportSectionTitles}>
-                {t('label.tested_positive_title')}
-              </Typography>
-              <Typography style={styles.exportSectionPara}>
-                {t('label.export_para_1')}
-              </Typography>
-              <Typography style={styles.exportSectionPara}>
-                {t('label.export_para_2')}
+              <Typography use='headline2' style={styles.exportSectionTitles}>
+                {t('share.title')}
               </Typography>
+              <Trans i18nKey='share.paragraphs'>
```

Comment:
Based on our convo lets remove this `<Trans>` and use a `t()` function in the Typography elements below.